### PR TITLE
 Good Clean Fun: A vending machine for games!

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Quiet areas of libraries on station have now been equipped with a
vending machine containing suitable recreational activities.
/:cl:

- Sprite by WJohn
- Removes dice bags and cards from the tables of libraries, you can now
get them from the vendor.
- If we ever implement checkers or go or chess, we should add the
playing kits to the vendor.

image

Box:
image

Delta:
image

Meta:
image

Omega:
image

Pubby:
image

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1178,3 +1178,15 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 #undef STANDARD_CHARGE
 #undef CONTRABAND_CHARGE
 #undef COIN_CHARGE
```
Comment (praisenarsie) [inline code/game/machinery/vending.dm]: move the undefs to the end of the file

**Comment by SpaceManiac** [comment]
I don't think you can over easels or photocopiers, so the Meta one looks inaccessible?

**Comment by coiax** [comment]
@SpaceManiac yeah, I have another PR to move that easel to art storage.

**Review by ShizCalev — CHANGES_REQUESTED** [review]
move undefs please.

**Pull request merged**